src/MissionManager/SpeedSection.cc

```diff
@@ -92,7 +92,7 @@ void SpeedSection::appendSectionItems(QList<MissionItem*>& items, QObject* missi
         MissionItem* item = new MissionItem(seqNum++,
                                             MAV_CMD_DO_CHANGE_SPEED,
                                             MAV_FRAME_MISSION,
-                                            _masterController->controllerVehicle()->multiRotor() ? 1 /* groundspeed */ : 0 /* airspeed */,    // Change airspeed or groundspeed
```

Comment:
The SpeedSection code is only used by a Waypoint ui item which also changes flight speed. (And in turn outputs both a waypoint item and a change speed item from the one plan element) Waypoint items don't support the new ascend/descend speed changes nor do I think they should. So this changes is not needed.